crx_template/wyrmhole.js

```diff
@@ -1,140 +1,80 @@
-/*global chrome*/
-(function() {
-
-
-    ////////////////////////////////////////
-    //  This script runs in the page!     //
-    ////////////////////////////////////////
-
-    var extId = "${PLUGIN_CRX_ID}";
-
+(function(fb) {
+	fb.wyrmhole = {
```

Comment:
When used in page context this will be problematic if multiple extensions are installed that support firebreath plugins, as each new one will override the old ones